util/build_configs/cray-internal/setenv-xe-x86_64.bash

```diff
@@ -175,7 +175,7 @@ if [ -z "$BUILD_CONFIGS_CALLBACK" ]; then
         log_info "Start build_configs $dry_run $verbose -- mason"
 
         $cwd/../build_configs.py $dry_run $verbose -s $cwd/$setenv -l "$project.mason.log" \
-            --target-compiler=gnu -- mason
+            --target-compiler=gnu -- mason-no-chpldoc
```

Comment:
I'd add a comment here about chpldoc not building on XE, but I'm okay with being overruled.  Otherwise, I think this is good to go!